docs/acme-divergences.md

```diff
@@ -4,61 +4,61 @@ While Boulder attempts to implement the ACME specification as strictly as possib
 
 This document details these differences, since ACME is not yet finalized it will be updated as numbered drafts are published.
 
-Current draft: [`draft-ietf-acme-acme-03`](https://tools.ietf.org/html/draft-ietf-acme-acme-03).
+Current draft: [`draft-ietf-acme-acme-04`](https://tools.ietf.org/html/draft-ietf-acme-acme-04).
 
-## [Section 5.5.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-5.5)
+## [Section 5.5.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-5.6)
 
 Boulder does not provide a `Retry-After` header when a user hits a rate-limit, nor does it provide `Link` headers to further documentation on rate-limiting.
 
-## [Section 5.6.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-5.6)
+## [Section 5.6.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-5.7)
 
 Boulder doesn't return errors under the `urn:ietf:params:acme:error:` namespace but instead uses the `urn:acme:error:` namespace from [draft-ietf-acme-01 Section 5.4](https://tools.ietf.org/html/draft-ietf-acme-acme-01#section-5.4).
 
 Boulder uses `invalidEmail` in place of the error `invalidContact` defined in [draft-ietf-acme-01 Section 5.4](https://tools.ietf.org/html/draft-ietf-acme-acme-01#section-5.4).
 
 Boulder does not implement the `caa` and `dnssec` errors.
 
-## [Section 6.1.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.1)
+## [Section 6.1.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-6.1)
 
 Boulder does not implement the `new-application` resource. Instead of `new-application` Boulder implements the `new-cert` resource that is defined in [draft-ietf-acme-02 Section 6.5](https://tools.ietf.org/html/draft-ietf-acme-acme-02#section-6.5).
```

Comment:
Similarly we'll need a new `Section 6.2` divergence for the "Getting a Nonce" section that was added describing using the `new-nonce` endpoint.